### PR TITLE
correct content alignment on all viewports

### LINKED ISSUE #1441: Ensure all content fits correctly on mobile viewports and devices
This might be a matter of iOS Safari but on certain pages, the Kitchensink for example, when you expand the menu and then click to focus on the input (so the iOS keyboard appears) the whole page shrinks a little and a white margin appears on the right. 

**Before focus**
IMG_4592

**After focus**
IMG_70571298572A-1

_Originally posted by @peterbe in https://github.com/mdn/yari/pull/1430#issuecomment-707782193_

### PR DESCRIPTION
**Pull request by schalkneethling**
Ensure that content always aligns properly at all viewports.
For now, until we have come to a conclusion on https://github.com/mdn/yari/issues/1486, test this without translated content so that the language switcher does not show up in the UI.

## Wide desktop

Screenshot 2020-10-20 at 13 27 46

## Most common desktop size (1366x768)

Screenshot 2020-10-20 at 13 30 43

## Tablet landscape

Screenshot 2020-10-20 at 13 31 14

## Tablet portrait

Screenshot 2020-10-20 at 13 31 04

## Mobile

Screenshot 2020-10-20 at 13 30 55

fix #1441

### CODE REVIEW

File: client/.storybook/preview.scss [hunk 1]
```diff
@@ -24,41 +25,43 @@ body {
...
+      background-color: #fffabc;
+    }
 
-  .page-footer {
-    background-color: #5e799a;
+    .page-footer {
+      background-color: #5e799a;
+    }
   }
 }
```
Comment (schalkneethling) [inline client/.storybook/preview.scss:67]: @peterbe you can safely ignore this file

File: client/src/stories/atoms/tables.stories.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
 export default {
-  title: "Docs/Tables",
+  title: "Atoms/Tables",
```
Comment (schalkneethling) [inline client/src/stories/atoms/tables.stories.js:4]: @peterbe you can safely ignore this file

File: client/src/stories/grid/reference-page-grid.stories.mdx [hunk 3]
```diff
@@ -4,33 +4,35 @@ import { Meta } from "@storybook/addon-docs/blocks";
...
+          <h1>Article</h1>
+        </article>
+      </main>
+      <nav class="sidebar">
+        <h1>Sidebar</h1>
+      </nav>
+    </div>
+    <footer class="page-footer">
+      <h1>Footer</h1>
+    </footer>
```
Comment (schalkneethling) [inline client/src/stories/grid/reference-page-grid.stories.mdx:36]: @peterbe you can safely ignore this file

File: client/src/stories/mocks/breadcrumbs.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  {
+    uri: "/en-US/docs/Web/HTML/Element",
+    title: "HTML elements reference",
+  },
+  {
+    uri: "/en-US/docs/Web/HTML/Element/audio",
+    title: "<audio>: The Embed Audio element",
+  },
+];
```
Comment (schalkneethling) [inline client/src/stories/mocks/breadcrumbs.js:18]: @peterbe you can safely ignore this file

File: client/src/stories/molecules/breadcrumb.stories.js [hunk 5]
```diff
@@ -3,33 +3,16 @@ import { BrowserRouter as Router } from "react-router-dom";
...
-    uri: "/en-US/docs/Web/HTML/Element/audio",
-    title: "<audio>: The Embed Audio element",
-  },
-];
-
 export const breadcrumbs = () => {
   return (
     <Router>
-      <Breadcrumbs parents={parents} />
+      <Breadcrumbs parents={breadcrumbParents} />
```
Comment (schalkneethling) [inline client/src/stories/molecules/breadcrumb.stories.js:15]: @peterbe you can safely ignore this file

File: client/src/stories/organisms/page-header.stories.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+          />
+        </div>
+        <div className="page-content-container">
+          <TOC toc={toc.toc} />
+        </div>
+      </div>
+    </Router>
+    <div className="page-overlay hidden"></div>
+  </>
+);
```
Comment (schalkneethling) [inline client/src/stories/organisms/page-header.stories.js:43]: @peterbe you can safely ignore this file

**Review by peterbe — APPROVED** [review]

**Pull request merged**